scripts/reboot_helper.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+#
+# reboot_helper.py
+#
+# Utility helper for reboot within SONiC
+
+import sonic_platform
+from sonic_py_common import logger
+from utilities_common.chassis import is_smartswitch, get_dpu_list
+
+SYSLOG_IDENTIFIER = "reboot_helper"
+
+EXIT_FAIL = -1
+EXIT_SUCCESS = 0
+
+# Global logger instance
+log = logger.Logger(SYSLOG_IDENTIFIER)
+
+# Global variable for platform chassis
+platform_chassis = None
```

Comment:
Do the following at once during init and don't need to keep checking the module loaded or not

```
  import sonic_platform.platform
   platform_chassis = sonic_platform.platform.Platform().get_chassis()
```

Reply:
@prgeor I have updated code to use the load_platform_chassis() from utilities_common/util_base.py.